ephios/core/models/users.py

```diff
@@ -117,6 +117,9 @@ class UserProfile(guardian.mixins.GuardianUserMixin, PermissionsMixin, AbstractB
     disabled_notifications = JSONField(
         default=list, encoder=CustomJSONEncoder, decoder=CustomJSONDecoder
     )
+    saved_questionnaire_answers = JSONField(
```

Comment:
will there be a view to view/(update)/delete saved answers for this PR? if not please create an issue for it

Reply:
Implemented in 1fddf194b7b6db7f3433c363926c6a3dd21fda1f